timescaledb/getting-started/index.md

```diff
@@ -1,23 +1,50 @@
+up your first instance of TimescaleDB, can connect to it, and are ready to 
+explore some of the most popular TimescaleDB feautures. If you have not yet 
+created a TimescaleDB instance or learned how to connect to it, make sure to 
+first check out these two sections within the docs:
+ * [Install TimescaleDB][install]
+ * [Connecting to TimescaleDB][connecting]
+
 
 ## What is TimescaleDB?
-Developers often refer to TimescaleDB as PostgreSQL with time-series superpowers.
-Put another way, TimescaleDB is a relational database with native time-series
-optimizations to make ingesting, storing and analyzing time-series data at petabyte
-scale fast and easy. TimescaleDB is packaged as a PostgreSQL extension. It also
-supports all the standard PostgreSQL database objects, like tables, indexes,
-triggers, and more. Moreover, a TimescaleDB database is able to coexist with
-PostgreSQL databases (and other TimescaleDB databases) on a single server.
-
-TimescaleDB also supports the full SQL language and you can use TimescaleDB with
-all the tools and connectors in the PostgreSQL ecosystem. If it works with
-PostgreSQL, it works with Timescale!
-
-## Let's get up and running
+As you likely know already, TimescaleDB is an extension on top of the PostgreSQL. 
+When you use TimescaleDB, you get all the time-series optimizations and special 
+functionality that TimescaleDB provides, along with all the features available 
+in vanilla PostgreSQL. Developers often refer to TimescaleDB as PostgreSQL with 
+superpowers.
+
+TimescaleDB supports the full SQL language and you can use TimescaleDB with
+all the tools and connectors within the PostgreSQL ecosystem. If it works with
+PostgreSQL, it works with TimescaleDB!
+
+## What you do in this Getting Started section
 This Getting Started section gives you a hands-on introduction to the 
-fundamentals of TimescaleDB. Using a real-world dataset, you'll learn definitions
-of key terms (like hypertables and chunks), mental models for working with
-TimescaleDB, and TimescaleDB's key features (like continuous aggregation,
-compression and data retention) and when, why and how to use them.
+fundamentals of TimescaleDB. You'll learn definitions
+of key terms like hypertables and chunks, and use some of TimescaleDB's key 
+features like continuous aggregation,
+data retention, and compression. 
+
+The data you ingest and use for this tutorial is real-time stock trading data 
+from the company [Twelve Data][twelve-data]. The dataset consists of two tables, 
+one with second-by-second stock-trade data for the top 100 most-traded symbols, 
+and the other containing company information which maps to the symbols.   
+
+As you work thorugh this tutorial and learn how to use some of TimescaleDB's most 
+popular features, keep in mind that there are many more you can explore further 
+throughout the docs. For more details on this check out the [next steps][next-steps] page. 
```

Comment:
I'd cut this paragraph and just put a happy "Alright... let's get started!!" paragraph/sentence. They'll get to "next steps" at the end.

Reply:
If you do keep the next steps link, remember to add the link to the bottom (it's not there right now) :)